explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala

```diff
@@ -72,6 +73,9 @@ object TargetSummaryTable {
     "epoch"        -> "Epoch",
     "pmra"         -> "µ RA",
     "pmdec"        -> "µ Dec",
+    "rv"           -> "RV",
```

Comment:
Do we want to display the three versions in the table?

Reply:
As per Andy's request, yes. We only show `RV` initially.